server/api/models.py

```diff
@@ -51,7 +55,78 @@ class Spell(models.Model):
     archetype = models.TextField()
     circles = models.TextField()
 
+class Archetype(models.Model):
+    slug = models.SlugField(unique=True, default=uuid.uuid1)
+    name = models.TextField(primary_key=True)
+    desc = models.TextField()
+
+class CharClass(models.Model):
+    slug = models.SlugField(unique=True, default=uuid.uuid1, primary_key=True)
+    name = models.TextField()
+    hit_dice = models.TextField()
+    hp_at_1st_level = models.TextField()
+    hp_at_higher_levels = models.TextField()
+    prof_armor = models.TextField()
+    prof_weapons = models.TextField()
+    prof_tools = models.TextField()
+    prof_saving_throws = models.TextField()
+    prof_skills = models.TextField()
+    equipment = models.TextField()
+    table = models.TextField()
+    desc = models.TextField()
+    spellcasting_ability = models.TextField()
+    subtypes_name = models.TextField()
+    archetype = models.ForeignKey(Archetype, on_delete=models.CASCADE, null=True)
```

Comment:
If there are going to be situations where a charclass has more than one archetype, then the relationship should be stored on the archetype - doing it this way, there can only be one archetype per charclass (since the foreign key is a single int).